02-oop-inheritance/script.js

```diff
@@ -0,0 +1,73 @@
+// 1 DONE- 4 - 8 DONE
+function Movie(title, year, duration) {
+	this.title = title;
+	this.year = year;
+	this.duration = duration;
+	this.cast = new Array();
+	this.play = function() {};
+	this.pause = function(){};
+	this.resume = function(){}
+	this.addCast = function(actors) {
+		if(Array.isArray(actors)) {
```

Comment:
You can make use of forEach method, let's see it:

```
var self = this;
actors.forEach(function (actor) {
   self.cast.push(actor);
});
```

Avoid the use of common `for` statement when dealing with collections, instead make use of collection methods like forEach, map, reduce, etc